### PR TITLE
Remove repeated code in vehicle.cpp

### PR DESCRIPTION
**Pull request by ghost**
A piece of code in vehicle.cpp is repeated twice.

### CODE REVIEW

File: src/vehicle.cpp [hunk 1]
```diff
@@ -3925,91 +3925,6 @@ void vehicle::operate_planter(){
...
-        } else {
-            const int speed = velocity;
-            const int v_damage = rng( 3, speed );
-            damage( plow_id, v_damage, DT_BASH, false );
-            sounds::sound( start_plow, v_damage, _("Clanggggg!") );
-        }
-    }
-}
-
-void vehicle::operate_reaper(){
```
Comment (BevapDin) [inline src/vehicle.cpp]: And you deleted the wrong version of those functions, those here are the good ones, pleases delete the other ones.

**Comment by chaosvolt** [comment]
Oh derp. So that's the cause of EVERY PR (except maybe 2 ancient ones) being wrecked? I assumed it was just merge conflicts via the PRs all being branched from a master predating the merge of the farming parts.

**Comment by DLaboratory** [comment]
> Also, seems #13652 also includes a fix for this. This'll be amusing to see how to handle both PRs.

Oops...

**Comment by chaosvolt** [comment]
Oh well, seems they can handle both PRs quite easily. XP

**Pull request merged**